internal/activity.go

```diff
@@ -146,6 +146,7 @@ type (
 
 		// RetryPolicy specify how to retry activity if error happens.
 		// Optional: default is to retry according to the default retry policy up to ScheduleToCloseTimeout
+		// with 50ms initial delay between retries and 2x backoff.
```

Comment:
The default value for RetryPolicy state the InitialInterval is 1s. I'm worried that this 50ms here would cause confusion.

Reply:
1 seconds seems too slow for local activity since the whole point is that they run locally and are mostly used for low latency use cases.

Reply:
Then we need to update the comment/doc for RetryPolicy.InitialInterval as well to reflect that it is 50ms for local activity. @mfateev

Reply:
Let's have a quick conversation about this tomorrow. I want to make sure we are all on the same page.

Reply:
Looking at java we have [1 second default](https://github.com/temporalio/sdk-java/blob/24a55be08d8cfaf0b8bb2bb49aab38e8344ce08c/temporal-sdk/src/main/java/io/temporal/common/RetryOptions.java#L232) there. This convinced me to make it the same for now.